### ISSUE TITLE
I can't see anything in License Check page in Sonarqube

### ISSUE THREAD

**Issue by seongilp** [porscheinformatik/sonarqube-licensecheck#374]
Hi, 
After running gradle command with sonar, I can't see anything in License Check page in Sonarqube.
`$ gradle sonar`
Is there anything that I can do?

**Comment by Janpopan** [comment]
Hi @seongilp can you provide some more context about your project structure?

**Comment by seongilp** [comment]
Hi, @Janpopan .
I've activated sonarqube license check plugin in my local environments.
After that, I edited build.gradle file and running command as below

```
sonarqube {
    properties {
        property "sonar.host.url", "http://localhost:9000"
    }
}
```


`> gradle sonarqube`

Above command was running successfully but I can't see anything in my sonar qube page.

**Comment by Janpopan** [comment]
Hi @seongilp what kind of project you try to analyse?

**Comment by Janpopan** [comment]
Did you read the[ readme.md](https://github.com/porscheinformatik/sonarqube-licensecheck#gradle)

**Comment by fvclaus** [comment]
@seongilp what version of SonarQube do you use? This plugin is currently broken with newer versions. I can't exactly tell you when it broke. It definitely doesn't work with 10.1. See #373 

One way to check is to open the DevTools and look for this message:
![image](https://github.com/porscheinformatik/sonarqube-licensecheck/assets/2384738/6df492c3-246c-4022-bd65-0abcdb3d3837)

That's the Vue.Js frontend being blocked.

There are also 404s in the log, but they don't cause issues.

**Comment by seongilp** [comment]
@fvclaus Hi, I'm using Community EditionVersion 10.0 (build 68432).

**Comment by fvclaus** [comment]
It won't work with 10.0. See https://github.com/porscheinformatik/sonarqube-licensecheck/pull/375#issuecomment-1606224896 for details

**Comment by seongilp** [comment]
@fvclaus Thanks! After merging your PR, is it works?

**Comment by fvclaus** [comment]
It should.

**Comment by derkoe** [comment]
Thx for @fvclaus this is fixed with #375

**Comment by derkoe** [comment]
Pre-release is out for that: [v6.0.0-alpha-1](https://github.com/porscheinformatik/sonarqube-licensecheck/releases/tag/v6.0.0-alpha-1)
